.pipelines/PowerShell-Release-Official.yml

```diff
@@ -0,0 +1,419 @@
+      policheck:
+        break: true # always break the build on policheck issues. You can disable it by setting to 'false'
+      # suppression:
+      #   suppressionFile: $(Build.SourcesDirectory)\.gdn\global.gdnsuppress
+      tsaOptionsFile: .config\tsaoptions.json
+
+    stages:
+    - stage: msixbundle
+      displayName: 'Create MSIX Bundle'
+      jobs:
+      - template: /.pipelines/templates/release-create-msix.yml@self
+
+    - stage: validateSdk
+      displayName: 'Validate SDK'
+      jobs:
+      - template: /.pipelines/templates/release-validate-sdk.yml@self
+        parameters:
+          jobName: "windowsSDK"
+          displayName: "Windows SDK Validation"
+          jobtype: windows
+
+      - template: /.pipelines/templates/release-validate-sdk.yml@self
+        parameters:
+          jobName: "MacOSSDK"
+          displayName: "MacOS SDK Validation"
+          jobtype: macos
+
+      - template: /.pipelines/templates/release-validate-sdk.yml@self
+        parameters:
+          jobName: "LinuxSDK"
+          displayName: "Linux SDK Validation"
+          jobtype: linux
+
+    - stage: gbltool
+      displayName: 'Validate Global tools'
+      jobs:
+      - template: /.pipelines/templates/release-validate-globaltools.yml@self
+        parameters:
+          jobName: "WindowsGlobalTools"
+          displayName: "Windows Global Tools Validation"
+          jobtype: windows
+
+      - template: /.pipelines/templates/release-validate-globaltools.yml@self
+        parameters:
+          jobName: "LinuxGlobalTools"
+          displayName: "Linux Global Tools Validation"
+          jobtype: linux
+          globalToolExeName: 'pwsh'
+          globalToolPackageName: 'PowerShell.Linux.x64'
+
+    - stage: fxdpackages
+      displayName: 'Validate FXD Packages'
+      jobs:
+      - template: /.pipelines/templates/release-validate-fxdpackages.yml@self
+        parameters:
+          jobName: 'winfxd'
+          displayName: 'Validate Win Fxd Packages'
+          jobtype: 'windows'
+          artifactName: 'drop_windows_package_package_win_fxdependent'
+          packageNamePattern: '**/*win-fxdependent.zip'
+
+      - template: /.pipelines/templates/release-validate-fxdpackages.yml@self
+        parameters:
+          jobName: 'winfxdDesktop'
+          displayName: 'Validate WinDesktop Fxd Packages'
+          jobtype: 'windows'
+          artifactName: 'drop_windows_package_package_win_fxdependentWinDesktop'
+          packageNamePattern: '**/*win-fxdependentwinDesktop.zip'
+
+      - template: /.pipelines/templates/release-validate-fxdpackages.yml@self
+        parameters:
+          jobName: 'linuxfxd'
+          displayName: 'Validate Linux Fxd Packages'
+          jobtype: 'linux'
+          artifactName: 'drop_linux_package_fxdependent'
+          packageNamePattern: '**/*linux-x64-fxdependent.tar.gz'
+
+      # defer task to run in Arm64
```

Comment:
Either add more comments about why this is still comment or remove.

Reply:
added a validation on arm64